src/main/java/org/triplea/client/ui/javafx/TripleA.java

```diff
@@ -0,0 +1,105 @@
+package org.triplea.client.ui.javafx;
```

Comment:
> I think I understand your reasoning here. ui is simply to tag the subpackages as containing pure UI classes for a specific framework?

Correct.

> I know it's not part of this PR, but if you can throw in some theoretical examples for the Swing client, as well, that might help to give me a better picture of how the client packages should be structured. I ask that because I'm wondering if you see client-specific code being shared between the Swing and JFX clients or if they are going to be totally separate (obviously they will both reference core "non-client" code, such as the engine, etc.).

I thought of a "SettingsPreferences" class, which is **_NOT_** relying on java.util.prefs.Preferences, but on a properties file or similar instead. This class could be "backported"/used by the current swing classes as well before the JavaFX GUI is finished